packages/react-examples/src/office-ui-fabric-react/DetailsList/DetailsList.Advanced.Example.tsx

```diff
@@ -722,7 +721,11 @@ export class DetailsListAdvancedExample extends React.Component<{}, IDetailsList
         column.isMultiline = true;
         column.minWidth = 200;
       } else if (column.key === 'name') {
-        column.onRender = (item: IExampleItem) => <Link data-selection-invoke={true}>{item.name}</Link>;
+        column.onRender = (item: IExampleItem) => (
+          <Link href="#" data-selection-invoke={true}>
```

Comment:
Should have caught this in master, but if you're setting href=# you should probably also add a click handler that does `ev.preventDefault()` to prevent jumping to the top of the page or other strange actions.

Reply:
The `href` is solely there for it to be a valid anchor and not for the functionality itself. Plus, we would probably need to inform the user why nothing happened when he clicked a link, no?
If you think it's pertinent, let's open a separate issue since the purpose of this PR was solely for cherry-picking into v7.

Reply:
I believe it will do the `onItemInvoked` action on click. Needing to have a click handler with preventDefault is a general issue with using `#` as `href`.

Reply:
Hmm actually something seems to be preventing the behavior of jumping to the top already, so I guess that's not an issue. Though I see what you mean about the action not being obvious (either to a screen reader user or sighted user) since it just logs to the console...

Reply:
How do you want to proceed on this?

Reply:
Sorry for not being clear. Let's just leave it as-is.